src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -776,19 +776,24 @@ static int do_map(int argc, const char *argv[])
 
 static int do_unmap()
 {
+  
+  int r = 0;
+
   int nbd = open_device(devpath.c_str());
   if (nbd < 0) {
     cerr << "rbd-nbd: failed to open device: " << devpath << std::endl;
     return nbd;
   }
 
-  if (ioctl(nbd, NBD_DISCONNECT) < 0) {
-    cerr << "rbd-nbd: the device is not used" << std::endl;
+  r = ioctl(nbd, NBD_DISCONNECT);
+  if (r < 0) {
+      r = -errno;
+      cerr << "rbd-nbd: disconnect failed: " << cpp_strerror(-r) << std::endl;
```

Comment:
@trociny , you are right，I have delete the empty line。and I have  modify commit message(add rbd-nbd:)， but about the NBD_DISCONNECT error message，I think the logic of disconnect is being executed，when disconnect ioctl happen error，use the error message rbd-nbd: disconnect failed and strerror(errno) can display the problem more obvious， for example，When the nbd device do disconnect by someone, the nbd device may be being used by others（by someone mount and so on）。thank you。